v-next/hardhat-ignition/src/index.ts

```diff
@@ -29,19 +29,19 @@ const hardhatIgnitionPlugin: HardhatPlugin = {
         type: ArgumentType.FILE,
         description:
           "A relative path to a JSON file to use for the module parameters",
-        defaultValue: "", // TODO: HH3 check this comes through correctly
+        defaultValue: "", // TODO: should this also be STRING_WITHOUT_DEFAULT?
```

Comment:
Lets resolve whether this arg should be a File.

Reply:
Currently it appears that the File and String types are treated the same. So some questions:

- Should we remove the File type
- Should this parameter just become a string
- Should there be a FILE_WITHOUT_DEFAULT

Reply:
We decided to add `FILE_WITHOUT_DEFAULT`